creator/settings/production.py

```diff
@@ -338,6 +338,7 @@ def traces_sampler(sampling_context):
 
 # This will be dynamically set depending on the file being requested
 AWS_S3_BUCKET_NAME = "kf-study-us-east-1-dev-sd-me0owme0w"
+AWS_S3_FILE_OVERWRITE = True
```

Comment:
I'm a bit confused..the docs say that the default for this setting is `True`. Does that mean that versions with the same name were getting overwritten in S3 before the change to prepend uuid?

Reply:
https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html

Reply:
Those are the docs for `django-storages`. We are using `django-s3-storage` where the default is `False`: https://github.com/etianen/django-s3-storage#file-storage-settings